packages/ssr-web/public/scripts/fix-vh.js

```diff
@@ -0,0 +1,25 @@
+(() => {
+  function supportsDvh() {
+    try {
+      return CSS.supports('(height: 1dvh)');
```

Comment:
dvh is Safari's upcoming fix for the vh problem. It's a viewport height that takes into account whether the url /bottom bar is showing.